dco_check/dco_check.py

```diff
@@ -29,7 +29,7 @@
 from urllib import request
 
 
-__version__ = '0.4.0'
+__version__ = '0.5.0'
```

Comment:
Please revert this; I usually bump the version separately when I make a release. I will of course create a new release after merging this PR.